src/AboutDialog.cpp

```diff
@@ -612,7 +612,10 @@ void AboutDialog::PopulateInformationPage( ShuttleGui & S )
 #endif
 
    // Location of settings
-   AddBuildinfoRow(&informationStr, XO("Settings folder:"), \
+   AddBuildinfoRow(&informationStr, XO("Config folder:"), \
```

Comment:
I think we could leave "Settings folder" here. What do you think, @Paul-Licameli?

Reply:
Okay.